platform-enterprise_versioned_docs/version-25.1/compute-envs/overview.md

```diff
@@ -28,9 +28,9 @@ Each compute environment must be configured to enable Seqera to submit tasks. Se
 
 If you have more than one compute environment, you can select a workspace primary compute environment to be used as the default when launching pipelines in that workspace. In a workspace, select **Compute Environments**. Then select **Make primary** from the options menu next to the compute environment you wish to use as default.
 
-## Edit compute environment
+## Rename compute environment
 
-From version 23.2, you can edit the names of compute environments in private and organization workspaces. Select **Edit** from the options menu next to the compute environment you wish to edit.
+From version 23.2, you can edit the names of compute environments in private and organization workspaces. Select **Rename** from the options menu next to the compute environment you wish to edit.
```

Comment:
```suggestion
You can edit the names of compute environments in private and organization workspaces. Select **Rename** from the options menu next to the compute environment you wish to edit.
```

Reply:
@llewellyn-sl, I've edited this based on a conversation last week around versioned docs not referring to previous versions in terms of functionality changes if the functionality is available in that version.